include/ignition/gazebo/rendering/Events.hh

```diff
@@ -65,6 +65,16 @@ namespace ignition
       /// \endcode
       using PostRender = ignition::common::EventT<void(void),
           struct PostRenderTag>;
+
+      /// \brief The force render event may be emitted outside the
+      /// rendering thread to force rendering calls.
```

Comment:
The approach looks reasonable to me.

I'd just improve the documentation a bit to explain that "forcing" rendering means that when the time comes, a plugin that performs rendering will do so even if it wasn't planning to.